src/AppBundle/Controller/Report/DocumentController.php

```diff
@@ -102,14 +103,20 @@ public function step1Action(Request $request, $reportId)
      */
     public function step2Action(Request $request, $reportId)
     {
-        $report = $this->getReportIfNotSubmitted($reportId, self::$jmsGroups);
-
-        if ( EntityDir\Report\Status::STATE_NOT_STARTED == $report->getStatus()->getDocumentsState()['state']
-           && 'yes' !== $report->getWishToProvideDocumentation()
```

Comment:
commenting here as not related to those changes: The ReportStatusService is normally used to tell the status, abstracting from what are the needed methods in the section (getWishToProvideDocumentation) in this case. No need to change, but to keep in mind and discuss in case we make future changes on those things

Reply:
In this case, if it's not started, the `getWishToProvideDocumentation`is supposed to be null, and there should be no need to check it. does it make sense ?

Reply:
Yes thats a nonsensical statement. Might as well be if (true && false)

Reply:
Removed impossible if statement above